packages/search-in-workspace/src/browser/search-in-workspace-preferences.ts

```diff
@@ -0,0 +1,48 @@
+/********************************************************************************
+ * Copyright (C) 2019 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { PreferenceSchema, PreferenceProxy, PreferenceService, createPreferenceProxy, PreferenceContribution } from '@theia/core/lib/browser/preferences';
+import { interfaces } from 'inversify';
+
+export const searchInWorkspacePreferencesSchema: PreferenceSchema = {
+    type: 'object',
+    properties: {
+        'search-in-workspace.lineNumbers': {
```

Comment:
please use VS Code preferences when it is possible:
https://github.com/Microsoft/vscode/blob/20cfb52ec124d15009244e9d0f6fb9e50a0d9948/src/vs/workbench/contrib/search/browser/search.contribution.ts#L710-L714